sympy/functions/elementary/miscellaneous.py

```diff
@@ -315,7 +315,7 @@ def real_root(arg, n=None):
         except ValueError:
             return root(arg, n)*Piecewise(
                 (S.One, ~Equality(im(arg), 0)),
-                (Pow(S.NegativeOne, S.One/n)**(2*floor(n/2)), And(
+                (Pow(S.NegativeOne, S.One/n, evaluate=evaluate)**(2*floor(n/2)), And(
```

Comment:
Line 312 above should be `rv = root(arg, n, evaluate=evaluate)` so that the evaluate argument is passed to root.